RHDEVDOCS-4303 - Added release notes for 1.4.11 and 1.5.5 (#48782)
openshift/openshift-docs: modules/gitops-release-notes-1-5-5.adoc

// Module included in the following assembly:
//
// * gitops/gitops-release-notes.adoc

:_content-type: REFERENCE

[id="gitops-release-notes-1-5-5_{context}"]
= Release notes for {gitops-title} 1.5.5

{gitops-title} 1.5.5 is now available on {product-title} 4.8, 4.9, and 4.10.

[id="new-features-1-5-5_{context}"]
== New features

The current release adds the following improvements:

* With this update, the bundled Argo CD has been updated to version 2.3.7.

[id="fixed-issues-1-5-5_{context}"]
== Fixed issues

The following issues have been resolved in the current release:

> * Before this update, the `redis-ha-haproxy` pods of an ArgoCD instance failed when more restrictive SCC's were in place in the cluster. This update fixes the issue by updating the security context in workloads. link:https://issues.redhat.com/browse/GITOPS-2034[GITOPS-2034]

Reviewer: ```suggestion
* Before this update, the `redis-ha-haproxy` pods of an ArgoCD instance failed when more restrictive SCCs were in the cluster. This update fixes the issue by updating the security context in workloads. link:https://issues.redhat.com/browse/GITOPS-2034[GITOPS-2034]
```

Reviewer: IBM SG: "Use an apostrophe and the letter s ('s) to form the plural of single letters." (but not multiple letters) For example: "Correct: The basic learning requirements for any industry are often known as the ABCs."

Reviewer: It looks like this change hasn't been made yet. How about:
```
... when more restrictive SCCs were present in the cluster....
```

[id="known-issues-1-5-5_{context}"]
== Known issues

> *  {gitops-title} Operator can make use of RHSSO (KeyCloak) through OIDC in addition to Dex. However, with a recent security fix applied, the certificate of RHSSO cannot be validated in some scenarios. link:https://issues.redhat.com/browse/GITOPS-2214[GITOPS-2214]

Reviewer: ```suggestion
*  {gitops-title} Operator can use RHSSO (KeyCloak) with OIDC and Dex. However, with a recent security fix applied, the Operator cannot validate the RHSSO certificate in some scenarios. link:https://issues.redhat.com/browse/GITOPS-2214[GITOPS-2214]
```
- Minimalism: make use of > use; in addition to > and; the certificate of RHSSO > the RHSSO certificate 
- Active voice: the Operator? cannot validate

Reviewer: If you adopt these changes, please apply them to the 1-4 release notes as well.

+
As a workaround, disable TLS validation for the OIDC (Keycloak/RHSSO) endpoint in the ArgoCD specification. 

> 

Reviewer: ```suggestion
+
```

Reviewer: Aligns the sample with the release note.

[source,yaml]
----

> spec:

Reviewer: Please add the following information, as indicated in
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields:
- Include the apiVersion and kind so that a user always knows the context of the YAML.
- Include the full hierarchy to a deeply nested key.

Reviewer: And do the same for the 1-4 file.

// Module included in the following assembly:
//
// * gitops/gitops-release-notes.adoc

:_content-type: REFERENCE

[id="gitops-release-notes-1-5-5_{context}"]
= Release notes for {gitops-title} 1.5.5

{gitops-title} 1.5.5 is now available on {product-title} 4.8, 4.9, and 4.10.

[id="new-features-1-5-5_{context}"]
== New features

The current release adds the following improvements:

* With this update, the bundled Argo CD has been updated to version 2.3.7.

[id="fixed-issues-1-5-5_{context}"]
== Fixed issues

The following issues have been resolved in the current release:

> * Before this update, the `redis-ha-haproxy` pods of an ArgoCD instance failed when more restrictive SCC's were in place in the cluster. This update fixes the issue by updating the security context in workloads. link:https://issues.redhat.com/browse/GITOPS-2034[GITOPS-2034]

Reviewer: ```suggestion
* Before this update, the `redis-ha-haproxy` pods of an ArgoCD instance failed when more restrictive SCCs were present in the cluster. This update fixes the issue by updating the security context in workloads. link:https://issues.redhat.com/browse/GITOPS-2034[GITOPS-2034]
```
